connect/json/src/test/java/org/apache/kafka/connect/json/JsonConverterTest.java

```diff
@@ -301,6 +301,15 @@ public void timestampToConnectOptional() {
         assertNull(schemaAndValue.value());
     }
 
+    @Test
+    public void timestampToConnectDefval() {
```

Comment:
Renamed the method to `logicalToConnectWithDefaultValue` - I think for testing this change needn't exercise all the logical types, one should be sufficient.